### PR TITLE
standardized implementation of images

### PR DESCRIPTION
**Pull request by sandro-elsweijer**
After reworking the introduction I thought it would be a good idea to adapt all images to the new format. Can be merged after #111

### CODE REVIEW

**Comment by joergbrech** [comment]
Thanks! I hate to be pedantic, but would you mind opening an issue associated to this change? I like issue-PR pairs to document the progress of the page and to understand later on why things were changed.

**Comment by sandro-elsweijer** [comment]
Closes  #114

**Comment by joergbrech** [comment]
This looks good, but I noticed that the TOC order is messed up again. We should hold of merging this until we figure out what is going on, see also #107 and #110.

**Comment by joergbrech** [comment]
@sandro-elsweijer, could you please rebase on/merge master? I want to see if the issue with the wrong order of chapters is resolved now that I merged #107.

**Comment by sandro-elsweijer** [comment]
> @sandro-elsweijer, could you please rebase on/merge master? I want to see if the issue with the wrong order of chapters is resolved now that I merged #107.

After merging the master branch the chapter seem to remain in the wrong order, at least on my local machine.

**Comment by joergbrech** [comment]
> > @sandro-elsweijer, could you please rebase on/merge master? I want to see if the issue with the wrong order of chapters is resolved now that I merged #107.
> 
> After merging the master branch the chapter seem to remain in the wrong order, at least on my local machine.


They look good in CI: https://998-168147770-gh.circle-artifacts.com/0/html/index.html. Or am I missing something?

**Comment by sandro-elsweijer** [comment]
> 
> 
> > > @sandro-elsweijer, could you please rebase on/merge master? I want to see if the issue with the wrong order of chapters is resolved now that I merged #107.
> > 
> > 
> > After merging the master branch the chapter seem to remain in the wrong order, at least on my local machine.
> 
> They look good in CI: https://998-168147770-gh.circle-artifacts.com/0/html/index.html. Or am I missing something?

I do not know why they have a wrong order on my local machine, but I can work with that. When the order is right in the CI, the order will probably be right on the website.

**Comment by joergbrech** [comment]
Ok, I am not super happy that the toc autocreation is so flaky. But then let's keep it for now and just be super diligent and double check the toc order in every PR.

File: content/00_einleitung/03_exercises/02_fluessigkeit_im_boden.ipynb [hunk 1]
```diff
@@ -9,7 +9,10 @@
     "Erstellen Sie ein Programm, in dem das Versickern einer Flüssigkeit im Boden in 2D modelliert wird. Legen Sie dazu eine Matrix an, die einen Bodenausschnitt mit **beliebiger** Breite $w$ und Tiefe $d$ darstellt. Die Oberfläche des Bodens sei im Bereich $[\\frac{w}{4},\\frac{3w}{4}]$ mit Flüssigkeit benetzt. \n",
     "\n",
     "<figure>\n",
-    "  <img src=\"images/fluessigkeit.png\" alt=\"Flüssigkeit\"/>\n",
+"```{image} images/fluessigkeit.png\n",
```
Comment (joergbrech) [inline content/00_einleitung/03_exercises/02_fluessigkeit_im_boden.ipynb:12]: This one doesn't get rendered correctly: https://998-168147770-gh.circle-artifacts.com/0/html/00_einleitung/03_exercises/02_fluessigkeit_im_boden.html

**Review by joergbrech — CHANGES_REQUESTED** [review]
The image on this page is missing: https://github.com/joergbrech/Modellbildung-und-Simulation/blob/master/content/11_pde/01_konvektion_und_diffusion.ipynb

**Comment by sandro-elsweijer** [comment]
> The image on this page is missing: https://github.com/joergbrech/Modellbildung-und-Simulation/blob/master/content/11_pde/01_konvektion_und_diffusion.ipynb

Do you mean that I missed this image in my standardization process or is there a second image in this chapter I cannot find?
If the first thing is the case I missed it, but I have changed it now.

**Comment by joergbrech** [comment]
LGTM! Thanks

**Review by joergbrech — APPROVED** [review]

**Pull request merged**